### PR TITLE
allow overriding pytest command

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,7 +18,8 @@ COLLECTION_COMMAND ?= ansible-galaxy
 SANITY_OPTS = --venv
 TEST =
 FLAGS =
-PYTEST = pytest -n 4 --forked -vv
+PYTEST_COMMAND ?= pytest -vv
```
Comment (ekohl) [inline Makefile]: Does it make sense to have one variable just for `pytest` and another for the options?

Note pytest natively already has `PYTEST_ADDOPTS` so you can use
```suggestion
PYTEST_COMMAND ?= pytest
PYTEST_ADDOPTS ?= -vv
```

Comment (evgeni) [reply Makefile]: I can never remember, are these then natively available in shells spawned by make, or do they need some kind of exporting?

Comment (ekohl) [reply Makefile]: That's a good question.

Comment (ekohl) [reply Makefile]: Submitted too early, but the answer is no:
```console
$ cat Makefile 
PYTEST_ADDOPTS ?= -vv

default:
	env
$ make | grep PYTEST
$ 
```

Comment (evgeni) [reply Makefile]: Then I don't think it gives us much benefit to split those up

Comment (ekohl) [reply Makefile]: To me the biggest benefit is that you don't need to think about `-v` or `-vv` when overriding it.

Comment (evgeni) [reply Makefile]: k, found a rather clean way to get what we both wanted :)

**Review by ekohl — APPROVED** [review]

**Pull request merged**